integration-tests/gradle-plugin-test/multi-platform/src/commonMain/kotlin/io/realm/test/multiplatform/model/TestClass.kt

```diff
@@ -0,0 +1,11 @@
+package io.realm.test.multiplatform.model
+
+import io.realm.kotlin.types.RealmObject
+import io.realm.kotlin.types.annotations.PrimaryKey
+
+class TestClass : RealmObject {
+    @PrimaryKey
+    var id: Long = 0
+
+    var text: String = "INIT"
+}
```

Comment:
Looks like we should also enable ktlint/detekt for these projects?

Reply:
Yep. Enabled our `realm-lint` plugin for this project too.